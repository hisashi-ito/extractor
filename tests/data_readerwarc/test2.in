WARC/1.0
WARC-Type: conversion
WARC-Target-URI: http://statmt.org/
WARC-Date: 2017-07-20T12:39:49Z
WARC-Record-ID: <urn:uuid:4e2c9e9d-16ad-487d-a7a3-a7844df236c2>
WARC-Refers-To: <urn:uuid:acf277dd-0b09-44bb-b7dd-1429e7b6cdca>
WARC-Block-Digest: sha1:UZJYKZZJH3QI3RMVNKBAHMU3VKEFZGS2
Content-Type: text/plain
Content-Length: 6598

This website is dedicated to research in statistical machine translation, i.e. the translation of text from one human language to another by a computer that learned how to translate from vast amounts of translated text.

WARC/1.0
WARC-Type: conversion
WARC-Date: 2017-07-20T12:39:49Z
WARC-Record-ID: <urn:uuid:4e2c9e9d-16ad-487d-a7a3-a7844df236c2>
WARC-Refers-To: <urn:uuid:acf277dd-0b09-44bb-b7dd-1429e7b6cdca>
Content-Type: text/plain
Content-Length: 6598

Moses is a statistical machine translation system that allows you to automatically train translation models for any language pair.
All you need is a collection of translated texts (parallel corpus).
Once you have a trained model, an efficient search algorithm quickly finds the highest probability translation among the exponential number of choices.

WARC/1.0
WARC-Type: conversion
WARC-Target-URI:http://www.fjoch.com/GIZA++.html
WARC-Date: 2017-07-20T12:39:49Z
WARC-Block-Digest: sha1:UZJYKZZJH3QI3RMVNKBAHMU3VKEFZGS2
Content-Type: text/plain
Content-Length: 6598

GIZA++ is an extension of the program GIZA (part of the SMT toolkit EGYPT) which was developed by the Statistical Machine Translation team during the summer workshop in 1999 at the Center for Language and Speech Processing at Johns-Hopkins University (CLSP/JHU).
GIZA++ includes a lot of additional features.
The extensions of GIZA++ were designed and written by Franz Josef Och.
