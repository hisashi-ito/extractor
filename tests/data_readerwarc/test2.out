df6fa1abb58549287111ba8d776733e9 uri:http://statmt.org/
This website is dedicated to research in statistical machine translation, i.e. the translation of text from one human language to another by a computer that learned how to translate from vast amounts of translated text.
df6fa1abb58549287111ba8d776733e9 uri:http://www.fjoch.com/GIZA++.html
GIZA++ is an extension of the program GIZA (part of the SMT toolkit EGYPT) which was developed by the Statistical Machine Translation team during the summer workshop in 1999 at the Center for Language and Speech Processing at Johns-Hopkins University (CLSP/JHU).
GIZA++ includes a lot of additional features.
The extensions of GIZA++ were designed and written by Franz Josef Och.
